src/transformers/keras_callbacks.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from tensorflow.keras.callbacks import Callback
+
+
+class PushToHubCallback(Callback):
+    def __init__(
+        self,
+        save_strategy="epoch",
```

Comment:
You can default to "epoch" but you should use the `IntervalStrategy` for this arg (and cast it from the string to that type). Also we need a type annotation
```suggestion
        save_strategy: Union[str, IntervalStrategy] = "epoch",
```

Reply:
Done!